Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/Utils.cpp

```diff
@@ -242,6 +242,10 @@ namespace ImageProcessingAtom
             }
         }
 
+        // Should we actually put the GetSubImagePixelValue API in this Utils file instead, since it already has
+        // some conversion logic, and has the helper method for retrieving an entire image (LoadImageFromImageAsset)
+        // whereas this is a helper for retrieving a specific pixel from the image?
+
```

Comment:
I didn't think any code from this gem was available at runtime?  We're going to need the code to exist in the runtime as well as the editor.

Reply:
Ah you're right good call, just checking! I'll remove this comment/question